third_party/kubeadm/app/constants/constants_unix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
we usually dont wanna change the third_party code, because they are copied in a simple way from upsream.

I suggest u add these changes to upstream kubernetes 
https://github.com/kubernetes/kubernetes/tree/master/cmd/kubeadm/app/constants

Reply:
The thing is they use `go 1.16` while we use `go 1.17` and those build tags don't exist in `1.16`, every time I run `go fmt` those files get changed.

Reply:
I see